--- klavaro-0.9.9.orig/src/tutor.c	2007-03-26 20:10:57.000000000 +0000
+++ klavaro-0.9.9/src/tutor.c	2007-03-26 20:12:19.000000000 +0000
@@ -5,6 +5,7 @@
 #include <math.h>
 #include <string.h>
 #include <time.h>
+#include <locale.h>
 
 #include "support.h"
 #include "interface.h"
--- klavaro-0.9.9.orig/src/plot.c	2007-03-26 20:10:57.000000000 +0000
+++ klavaro-0.9.9/src/plot.c	2007-03-26 20:14:32.000000000 +0000
@@ -2,6 +2,7 @@
  * Charts management
  */
 
+#include <locale.h>
 #include "gtkdatabox.h"
 #include "support.h"
 #include "main.h"
